.test-infra/terraform/google-cloud-platform/google-kubernetes-engine/variables.tf

```diff
@@ -31,11 +31,27 @@ variable "cluster_name_prefix" {
   description = "The prefix to assign the provisioned Google Kubernetes Engine (GKE) cluster; a random string is appended to this value"
 }
 
+variable "cluster_name_override" {
+  type = string
+  description = "Use this to override naming and omit the postfix. Leave empty to use prefix-suffix format"
+  default = ""
+}
+
```

Comment:
Could we remove this variable and just have the prefix to keep it simple?

Reply:
Why not just keep the kafka cluster running continually and delete the topics after the workflows execute?

Reply:
We've had flakey tests in this repository due to waiting on spinning up new clusters.

Reply:
This way we ensure its fresh each time which is easier then to maintain a kafka instance and make sure it does not break between different tests. We could delete topics but still there could be issues.